packaging/keybase.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/sh
+
+# This creates a Keybase release on github from the current source/tagged version.
+
+set -e # Fail on error
+
+dir=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+cd $dir
+
+build_dir=$dir/build_keybase
+
+clientdir="$GOPATH/src/github.com/keybase/client"
+
+# Ensure release tool
+go install github.com/keybase/client/go/tools/release
+
+version=`release version`
+tag="v$version"
+tgz="keybase-$version.tgz"
+
+if [ "$GITHUB_TOKEN" = "" ]; then
```

Comment:
If we add `set -u` to the top, `"$GITHUB_TOKEN"` will need to become `"${GITHUB_TOKEN:-}"` (I think).